ormlitegenerator/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
no different than GreenDAO...it essentially requires a compilation step to get rid of the reflection